_docs/samples/bookinfo.md

```diff
@@ -35,23 +44,26 @@ The end-to-end architecture of the application is shown below.
 
 This application is polyglot, i.e., the microservices are written in different languages.
 
-1. Follow the Istio [Installation]({{site.bareurl}}/docs/tasks/istio-installation.html) guide.
-
 ## Start the Application
+
 1. Change your current working directory to the bookinfo application directory:
```

Comment:
@andraxylia @frankbu I am thinking that this bit could be repurposed for "Integrating your application with Istio".  Thoughts?